### ISSUE TITLE
Modrinht version

### ISSUE THREAD

**Issue by Simon11010** [Bl4ues/SCP-Additions#2]
I heard there is a modified version of this mod that is being updated. Is there actually a modrinht version (if so could I get a link) or has the project been abandoned?

**Comment by Bl4ues** [comment]
Sorry for the delay. I was planning on it before, when I was engaged in the production of the mod. Update 1.0.7 had a loooot of features and updates and I really didn't manage to get everything where I wanted because two SCPs that I chose to make were... ambitious, to say the least.
But now, while at university, I had a moment of "you know what?" and decided to post the update the way it was when I left it, even if some things are still WIP. I just now submited the project on Modrinth and will be updating it there from now on, just not as frequently as before.

I'm still awaiting review, but the link will be this one: https://modrinth.com/mod/scp-additions

Thank you for reaching the project! It's not abandoned yet, I hope xd
